README.md

```diff
@@ -43,6 +43,9 @@ Possible options are:
 
 For more details on how semantic version difference is calculated please see [semver](https://www.npmjs.com/package/semver) package.
 
+If you set a value other than `any`, PRs that are not semantic version compliant are skipped.
+An example of a non-semantic version is a commit hash.
```

Comment:
looks good, can we add just one piece to this sentence to clarify when this may happen? e.g. : when using git submodules